spring-components/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.rdf4j</groupId>
+		<artifactId>rdf4j</artifactId>
+		<version>4.0.0-SNAPSHOT</version>
+	</parent>
+	<packaging>pom</packaging>
+	<modules>
+		<module>spring-boot-sparql-web</module>
+	</modules>
+	<properties>
+		<spring.boot.version>2.4.3</spring.boot.version>
+	</properties>
+	<artifactId>rdf4j-spring-boot-components</artifactId>
```

Comment:
This should be `rdf4j-spring-components`. And I suggest that for now we stick with that as the name rather than the alternatives we discussed. Let's see what it looks like after @fkleedorfer has started their followup PR to add another submodule in, it's probably easier to adjust the naming after that.